target/linux/ipq40xx/image/Makefile

```diff
@@ -221,6 +221,30 @@ define Device/qcom_ap-dk04.1-c1
 endef
 TARGET_DEVICES += qcom_ap-dk04.1-c1
 
+define Device/qxwlan_e2600ac-c1
+	$(call Device/FitImage)
+	DEVICE_TITLE :=  Qxwlan E2600AC-C1 (nor flash)
+	BOARD_NAME := e2600ac-c1
+	DEVICE_DTS := qcom-ipq4019-qxwlan-e2600ac-c1
+	KERNEL_SIZE := 4096k
+	IMAGE_SIZE := 26624k
+	IMAGES := sysupgrade.bin
+	IMAGE/sysupgrade.bin := append-kernel |append-rootfs | pad-rootfs | append-metadata
+endef
+TARGET_DEVICES += qxwlan_e2600ac-c1
+
+define Device/qxwlan_e2600ac-c2
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_DTS := qcom-ipq4019-qxwlan-e2600ac-c2
+	KERNEL_INSTALL := 1
+	KERNEL_SIZE := 4096k
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	DEVICE_TITLE := Qxwlan E2600AC-C2 (nornand flash)
+endef
```

Comment:
Please provide a package for your wifi's boardfiles and then add it with
DEVICE_PACKAGES := ipq-wifi-qxwlan_e2600ac